src/main/java/roomescape/domain/Reservation.java

```diff
@@ -1,19 +1,20 @@
 package roomescape.domain;
 
 public class Reservation {
-    Long id;
-    String name;
-    String date;
-    String time;
+    private Long id;
+    private String name;
+    private String date;
+    private Time time;
 
     public Reservation() {
+        time = new Time();
     }
 
-    public Reservation(Long id, String name, String date, String time) {
+    public Reservation(Long id, String name, String date, Long timeId, String timeValue) {
```

Comment:
외부에서 Time 객체를 만들어서 내부로 매핑해주는건 어떤가요?

```suggestion
    public Reservation(Long id, String name, String date, Time time) {
```